aup.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Acceptable Use Policy"
+---
+
+The terms "R-multiverse", "User", "Contributor", "Moderator" and "Administrator" shall have the meaning as defined in the [R-multiverse Governance Document](governance.md).
+This Acceptable Use Policy (the "Policy") outlines the standards of conduct expected from all Users and Contributors to R-multiverse (the "Repository").
+By accessing or using the Repository, you agree to comply with this Policy.
+Violations of this Policy may result in the suspension or termination of your access to the Repository.
+
+## 1. Prohibited Content
+
+Users and Contributors must not submit, post, or share any content that:
+
+- **Illegal or Unlawful**: Violates any applicable laws, regulations, or guidelines in your jurisdiction, including content that promotes illegal activities, constitutes fraud, or violates intellectual property rights.
+- **Malicious Software**: Contains viruses, malware, ransomware, trojan horses, or any other harmful or malicious code designed to disrupt, damage, or interfere with the proper functioning of any software, hardware, or network.
+- **Harassment and Hate Speech**: Includes language or content that is abusive, defamatory, harassing, discriminatory, threatening, or hateful based on race, gender, ethnicity, religion, disability, sexual orientation, or any other protected characteristic.
+- **Spam and Unsolicited Messages**: Involves sending unsolicited messages, advertising, promotional materials, or any form of spam not directly related to the technical content of the Repository.
+
+## 2. Prohibited Activities
+
+Users and Contributors are prohibited from engaging in activities that:
+
+- **Unauthorized Access**: Attempt to gain unauthorized access to the Repository or any associated systems, networks, or data.
+- **Disruption of Services**: Interfere with or disrupt the operation of the Repository, including launching denial-of-service attacks or other harmful activities.
+- **Circumventing Security Measures**: Attempt to bypass or disable any security or authentication measures implemented by the Repository administrators.
+- **Data Harvesting**: Use any automated system, including "bots", "scrapers", or other similar data-gathering tools, to extract data from the Repository for unauthorized purposes.
```

Comment:
so no analysis of code?

Reply:
I thought about that as well. The reason I kept it in is there's an exception for "for unauthorized purposes". So I'm thinking if we provide APIs for example that would automatically be allowed. But just extra protection against mass harvesting which may cause us to burn through data hosting transfer quotas etc.